tests/migration/migration.go

```diff
@@ -2725,7 +2725,12 @@ var _ = SIGMigrationDescribe("VM Live Migration", decorators.RequiresTwoSchedula
 
 			By("Creating ResourceQuota with enough memory for the vmi but not enough for migration")
 			resourceQuota := newResourceQuota(resourcesToLimit, testsuite.GetTestNamespace(vmi))
-			_ = createResourceQuota(resourceQuota)
+			resourceQuota = createResourceQuota(resourceQuota)
+			Eventually(func(g gomegatypes.Gomega) error {
+				quota, err := virtClient.CoreV1().ResourceQuotas(resourceQuota.Namespace).Get(context.TODO(), resourceQuota.Name, metav1.GetOptions{})
+				g.Expect(quota.Status.Hard).To(Equal(resourcesToLimit))
+				return err
+			}).WithTimeout(time.Minute).WithPolling(time.Second).Should(Succeed())
```

Comment:
you might want to look into  `resourceQuota.Status.Used` and use the same function as Kubernetes:
https://github.com/kubernetes/kubernetes/blob/566f939b19fa0be3d3f5d11b13c38884887eb915/test/e2e/apimachinery/resource_quota.go#L2215

Reply:
I think it the same for us, we just wait for the quota to start functioning...

Reply:
I think that I tried this with AAQ, but the tests were still flaky until I looked into the `Status.Used`, but I'm not really sure, it was a long time ago.

Reply:
Thanks, I am going to check for both then.